docs/cql/dml.rst

```diff
@@ -601,7 +601,7 @@ by the same node are unique. Timestamps assigned at different
 nodes are not guaranteed to be globally unique.
 With a steadily high write rate timestamp collision
 is not unlikely. If it happens, i.e. two ``INSERTS`` have the same
-timestamp, the lexicographically bigger value prevails:
+timestamp, a tie-breaking algorithm determines which of the inserts prevails (see :ref:`update ordering <update-ordering>`).
```

Comment:
System behavior **must** be documented. Period! There is nothing worse and "undefined behavior".
What users should or should not do - leave to users to decide.
If you are personally so against some particular practices you are welcome to write a "Best Practices" article and reference it from the document page mentioned above. 
I suggest we continue this discussion on the PR that adds this article.

Reply:
I can drop the patch that updates the documentation from this series if it's still controversial.
I think it's okay to document our behavior, but we should still be careful to be clear about what we want and can guarantee v.s. what we consider as implementation details that are subject to change, and also timing-sensitive so may produce different result due to server time-skew and/or low resolution of ttl - and so we advise the users not to step into those grey areas.

Reply:
Added disclaimer in v6

Reply:
What about 

> What if someone relied on "the lexicographically bigger value prevails"?

The old behavior was documented, so a user could have "decided" to rely on it.

Reply:
I see your point, but what exactly do you think we should do?

Reply:
I agree with @tgrabiec, I think we can document that if a user does two writes with the same timestamp, the result MAY be a mixed version of the two writes - without explain exactly which extenuating circumstances (such as TTL and conflicting writes more than one second apart) may ensure that the conflict will result in one of the two writes but not n a mix.

I'd love to have a developer-oriented documentation (in docs/dev/) for the conflict resolution details, but I don't think users should make use of these details (and if they did, now with this PR, they will be frustrated because the details changed).